generators/random_signal.py

```diff
@@ -0,0 +1,17 @@
+import random
+from nio import Signal
+
+
+class RandomSignalGenerator():
+
+    def generate_signals(self, n=1):
+        n = random.randint(0, 4)
```

Comment:
Let's make these min and max configurable

Reply:
Oh wait, I just realized this is a random number of signals being output, not a random number on a signal. The generators shouldn't be able to control how many signals are generated. They are told how many signals to generate and they do it.

If you wanted a random number of signals generated I would think that would be a random trigger instead. Basically, randomly trigger the generation of the next signal. So you could randomly generate the next number in a count, randomly generate another identity signal, or randomly generate a random number.

Reply:
That makes sense, it felt a little weird that `generate_signals` takes an argument `n` that is over-ridden immediately.

I agree that the randomness should have min/max configuration.